rules/linux/persistence_suspicious_ssh_execution_xzbackdoor.toml

```diff
@@ -0,0 +1,102 @@
+[metadata]
+creation_date = "2024/04/01"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/04/01"
+
+
+[rule]
+author = ["Elastic"]
+description = """
+It identifies potential malicious shell executions through remote SSH and detects cases where the sshd service suddenly
+terminates soon after successful execution, suggesting suspicious behavior similar to the XZ backdoor.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Execution via XZBackdoor"
+references =["https://github.com/amlweems/xzbot", "https://access.redhat.com/security/cve/CVE-2024-3094"]
+risk_score = 73
+rule_id = "7afc6cc9-8800-4c7f-be6b-b688d2dea248"
+severity = "high"
+tags = [
+        "Domain: Endpoint",
+        "OS: Linux",
+        "Use Case: Threat Detection",
+        "Tactic: Credential Access",
+        "Tactic: Persistence",
+        "Tactic: Lateral Movement",
+        "Data Source: Elastic Defend"
+        ]
+timestamp_override = "event.ingested"
+type = "eql"
+query = '''
+sequence by host.id with maxspan=1s
+ [process where host.os.type == "linux" and event.action == "exec" and process.name == "sshd"] by process.pid, process.entity_id
+ [process where host.os.type == "linux" and event.action == "exec" and process.name == "sh" and process.parent.name == "sshd"] by process.parent.pid, process.parent.entity_id
+ [process where host.os.type == "linux" and event.action == "end" and process.name == "sshd" and process.exit_code == 255] by process.pid, process.entity_id
```

Comment:
I assume this rule is attempting to catch connections to the backdoor itself based on process tree anomalies depicted in the reference?

Is the last network disconnect sequence necessary?

Reply:
> I assume this rule is attempting to catch connections to the backdoor itself based on process tree anomalies depicted in the reference?

based on testing using xzbot

> Is the last network disconnect sequence necessary?

in prod env yes, to match on SSH servers facing internet

Reply:
![image](https://github.com/elastic/detection-rules/assets/64742097/1367a857-b0b9-4fe4-b6c9-ca442f9fe97f)